keyboards/telophase/telophase.h

```diff
@@ -46,15 +46,16 @@
 // The first section contains all of the arguements
 // The second converts the arguments into a two-dimensional array
 #define LAYOUT( \
-  k00, k01, k02, k03, k04, k05,      k06, k07, k08, k09, k10, k11, \
-  k12, k13, k14, k15, k16, k17,      k18, k19, k20, k21, k22, k23, \
-  k24, k25, k26, k27, k28, k29,      k30, k31, k32, k33, k34, k35, \
-  k36, k37, k38, k39, k40, k41,      k42, k43, k44, k45, k46, k47  \
+  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k10, k11, \
+  k12, k13, k14, k15, k16, k17, k18, k19, k20, k21, k22, k23, \
+  k24, k25, k26, k27, k28, k29, k30, k31, k32, k33, k34, k35, \
+  k36, k37, k38, k39, k40, k41, k42, k43, k44, k45, k46, k47  \
 ) \
 { \
-  { k11, k10, k09, k08, k07, k06, KC_NO, KC_NO, k05, k04, k03, k02, k01, k00},  \
-  { k23, k22, k21, k20, k19, k18, KC_NO, KC_NO, k17, k16, k15, k14, k13, k12},  \
-  { k35, k34, k33, k32, k31, k30, KC_NO, KC_NO, k29, k28, k27, k26, k25, k24},  \
-  { k47, k46, k45, k44, k43, k42, KC_NO, KC_NO, k41, k40, k39, k38, k37, k36},  \
-  { KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO,KC_NO}  \
```

Comment:
These changes were specific to my use-case as I wired my keyboard up backwards. Is there an appropriate place to put my specific `LAYOUT` macro? somewhere in the `/mathieu` folder?

Reply:
I believe you can just add it to telophase.h as `LAYOUT_reverse` or something similar.